File: exist/ringo/read/exist_read.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	exist_read.js
//
//						May/09/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
var xml_manipulate = require ('/var/www/data_base/common/ringo_common/xml_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");

var client= require ("ringo/httpclient");

var url='http://localhost:8086/exist/rest/db/cities/cities.xml';

var data=client.get (url);

print (data.status);
// print (data.content);

var dict_aa = xml_manipulate.xml_to_dict_proc (data.content);

text_manipulate.dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: exist/jruby/read/exist_read.rb
#! /usr/bin/jruby
#
#	exist_read.rb
#
#					May/19/2011
#
require "rexml/document"
require	"net/http"
load '/var/www/data_base/common/ruby_common/xml_manipulate.rb'
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
include REXML
#
# -----------------------------------------------------------------
print	"*** 開始 ***\n"
#
host= "localhost"
port = 8086
path="/exist/rest/cities/cities.xml"
#
Net::HTTP.start(host,port)do |http|
		contents=http.get(path)
#		print contents.body
		dict_aa=xml_to_dict_proc(contents.body)
		dict_display_proc(dict_aa)
	end
#
print	"*** 終了 ***\n"
# -----------------------------------------------------------------
